hatchways/team-cosmic-crisp
# FS setup socketio

### What this PR does (required):
- Socket deps have already been installed
- Socket server is setup
- Added auth to sockets handshake

### Screenshots / Videos (required):

![2](https://user-images.githubusercontent.com/75184965/120372149-a22deb00-c2cb-11eb-8ffd-b5b8f724963e.PNG)

![Capture](https://user-images.githubusercontent.com/75184965/120372197-b376f780-c2cb-11eb-85d5-19b1b5dbc01e.PNG)


### Any information needed to test this feature (required):
- `npm start` on client | `npm run dev` on server
- Login/Sign Up
- Check server console to see message being logged when user is logged in.

### Any issues with the current functionality (optional):
- None
